PhysicsTools/Heppy/interface/TriggerBitChecker.h

```diff
@@ -27,9 +27,9 @@ namespace heppy {
     bool check_unprescaled(const edm::EventBase &event,
                            const edm::TriggerResults &result_tr,
                            const pat::PackedTriggerPrescales &result) const;
-    int getprescale(const edm::EventBase &event,
-                    const edm::TriggerResults &result_tr,
-                    const pat::PackedTriggerPrescales &result) const;
+    double getprescale(const edm::EventBase &event,
```

Comment:
do we wish to consider protecting this against people containing to call this expecting an int via the template with default to int and static assert for having an double/FracPrescale